server/src/api/context.go

```diff
@@ -44,21 +44,55 @@ func (s *Server) BoardParticipantContext(next http.Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		log := logger.FromRequest(r)
 
+		var board uuid.UUID
 		boardParam := chi.URLParam(r, "id")
-		board, err := uuid.Parse(boardParam)
-		if err != nil {
-			common.Throw(w, r, common.BadRequestError(errors.New("invalid board id")))
+		if boardParam != "" {
+			var err error
+			board, err = uuid.Parse(boardParam)
+			if err != nil {
+				common.Throw(w, r, common.BadRequestError(errors.New("invalid board id")))
+				return
+			}
+		} else {
+			// board id is not defined in the url but maybe in the context
```

Comment:
Is there any endpoints where that's the case?

Reply:
I agree, that is an unnecessary check